### PR TITLE
Use tar(1) for compressing packages

### PR DESCRIPTION
**Pull request by sonicaj**
This commit introduces a change where we make use of tar(1) for compressing packages instead of tarfile because tarfile is unable to preserve file flags.

### CODE REVIEW

File: create_package/create_package.py [hunk 1]
```diff
@@ -421,40 +419,48 @@ def matches(path, pattern):
 
     # I would LOVE to be able to use xz, but python's tarfile does not
```
Comment (ghost) [inline create_package/create_package.py]: This comment looks irrelevant now (and should we consider using xz?).

Comment (sonicaj) [reply create_package/create_package.py]: It has been for quite a while, we switched to python3 a long time ago :P However, i think we are good for now and can focus on more important stuff ; ) Thoughts @william-gr ?

**Review by william-gr — APPROVED** [review]
